htdocs/compta/bank/releve.php

```diff
@@ -352,7 +354,7 @@
 		$massactionbutton = $form->selectMassAction('', $arrayofmassactions);
 
 		$morehtml = '';
-		if ($action != 'addline' && $action != 'reconcile') {
+		if ($action != 'addline' && $action != 'reconcile' && $buttonreconcile) {
```

Comment:
I would just move this part into the code section that was commented out.
That is what made this error appear and now makes this code "useless".
However, move $morehtml to the location where `$buttonreconcile = ''` that no longer needs to be added.